package.json

```diff
@@ -473,6 +473,12 @@
           "default": true,
           "scope": "resource"
         },
+        "mssql.saveAsCsv.delimiter": {
+          "type": "string",
+          "description": "%mssql.saveAsCsv.delimiter%",
+          "default": true,
```

Comment:
Default needs to be updated to ","

Reply:
Updated with corrected value. Also, verified locally with gulp package, it is working fine.